be/src/exec/local_file_reader.cpp

```diff
@@ -53,21 +53,17 @@ bool LocalFileReader::closed() {
 }
 
 // Read all bytes
-Status LocalFileReader::read_one_message(uint8_t** buf, size_t* length) {
+Status LocalFileReader::read_one_message(std::unique_ptr<uint8_t[]>& buf, size_t* length) {
```

Comment:
Use pointer if the parameter will be changed.
Use reference with const 
```suggestion
Status LocalFileReader::read_one_message(std::unique_ptr<uint8_t[]>* buf, size_t* length) {
```

Reply:
I will apply it